README.md

```diff
@@ -40,7 +40,7 @@ ports:
 artifacts:
 - name: "DECISION_CENTRAL_DISTRIBUTION_ZIP"
   target: "decision_central_distribution.zip"
-  # rhdm-7.8.0.redhat-20200224-decision-central-eap7-deployable.zip
+  # rhdm-7.8.1.redhat-20200224-decision-central-eap7-deployable.zip
   md5: "759fc7de8cdc5f366e90bdd10c66346e"
```

Comment:
Should this be :

```suggestion
  md5: "704987c76eb53b351a1816cf62b18486"
```

Reply:
That's correct, forgot to update both, thanks!